pages/platform/ai/training_tuto_42_models_comparaison_weights_and_biases/guide.en-us.md

```diff
@@ -0,0 +1,488 @@
+
+To know more about **Weights & Biases**, please refer to the [documentation](https://docs.wandb.ai/?_gl=1*pk7t4x*_ga*MTgyOTU1NDM2My4xNjU4OTk2NzQy*_ga_JH1SJHJQXJ*MTY1OTM2MDMxNi45LjEuMTY1OTM2MTI3Ny41OQ..).
+
+The basic principles for using Weights & Biases can be found [here](https://github.com/ovh/ai-training-examples/blob/main/notebooks/computer-vision/image-classification/tensorflow/weights-and-biases/notebook_Weights_and_Biases_MNIST.ipynb) with AI Notebooks.
+
+## Requirements
+
+- Access to the [OVHcloud Control Panel](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/world/&ovhSubsidiary=we);
+- An AI Training project created inside a [Public Cloud project](https://www.ovhcloud.com/en/public-cloud/) in your OVHcloud account;
+- A [user for AI Training](https://docs.ovh.com/us/en/publiccloud/ai/users/);
+- [Docker](https://www.docker.com/get-started) installed on your local computer;
+- Make sure you have a Docker Hub [account](https://hub.docker.com/);
+- Some knowledge about building image and [Dockerfile](https://docs.docker.com/engine/reference/builder/);
+- A Weights & Biases account, you can create it on their [website](https://wandb.ai/site). *It’s Free for individuals.*
+
+## Instructions
+
+You will follow different steps to process your data and train your two models.
+
+- More detailed data processing in this [notebook](https://github.com/ovh/ai-training-examples/blob/main/notebooks/audio/audio-classification/notebook-marine-sound-classification.ipynb) concerning the classification of marine mammal sounds.
+- Direct link to the full Python files can be found here [here](https://github.com/ovh/ai-training-examples/tree/main/jobs/weights-and-biases/audio-classification-models-comparaison).
+
+The tutorial is as follows:
+
+![image](images/instructions-overview.jpeg){.thumbnail}
+
+Here we will mainly discuss how to write the data processing and models training codes, the `requirements.txt` and `packages.txt` files and the `Dockerfile`. If you want to see the whole code, please refer to the [GitHub repository](https://github.com/ovh/ai-training-examples/tree/main/jobs/weights-and-biases/audio-classification-models-comparaison).
+
+### Clone the GitHub repository
+
+The first thing to do is to clone the GitHub repository.
+
+```console
+git clone https://github.com/ovh/ai-training-examples
+```
+
+You can then place yourself in the dedicated directory.
+
+```console
+cd ai-training-examples/jobs/weights-and-biases/audio-classification-models-comparaison
+```
+
+### Uploading your dataset on Public Cloud Storage
+
+> [!primary]
+>
+> First, download the data on [Kaggle](https://www.kaggle.com/datasets/subhajournal/free-spoken-digit-database).
+>
+
+It's a zip file (`audio_files.zip`)! Push it into an object container named `spoken-digit`.
```

Comment:
here its not clear that you should continue reading to know how to push an object. 
At first I was like : "and if I do not know how to do this, i'm stuck" :-) 

something like "we are going to push it"